src/scripts.js

```diff
@@ -193,5 +234,6 @@ function displaySleepData() {
 
 function returnUserSelectedDate() {
   let test = dayjs(date.value).format('YYYY/MM/DD')
+  console.log("dayjs", test);
```

Comment:
Let's be sure we don't leave any console logs anywhere when we get into refactoring